example/plugin/main.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+/*
+ * SPDX-FileCopyrightText: 2023 Siemens AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Author: Michael Adler <michael.adler@siemens.com>
+ */
+
+import (
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/siemens/wfx/generated/plugin"
+	"github.com/siemens/wfx/generated/plugin/client"
+	"github.com/siemens/wfx/middleware/plugin/ioutil"
+)
+
+const queueSize = 64
+
+type QueueEntry struct {
+	start   time.Time
+	request *plugin.PluginRequestT
+}
+
+func main() {
+	// NOTE: The working directory for this process is inherited from the wfx parent process
```

Comment:
Maybe this note should also be put into the documentation? Opening log files in this directory may not always be possible if wfx is running from `/usr`, for example.

Reply:
Yep added

Reply:
Hm, the documentation doesn't mention the creation of `plugin.log` and that this directory needs to be writable?

Reply:
Yep I've switched to stderr now

Reply:
For an example, that's OK though....